app/src/main/java/com/android/streetworkapp/ui/park/ParkOverview.kt

```diff
@@ -388,8 +390,11 @@ fun InteractiveRatingComponent(rating: MutableState<Int>) {
  * @param navigationActions The navigation actions to navigate to other screens.
  */
 @Composable
-fun EventItemList(eventViewModel: EventViewModel, navigationActions: NavigationActions) {
-  val uiState = eventViewModel.uiState.collectAsState().value
```

Comment:
I think this might be an error du to merges but the `uiState` in the parkOverview was declared directly inside the `EventItemList` because it is the only place where it is used, as such i would revert this change if it is  alright with you (i.e remove the uiState from the `EventItemList` and revert the above line)

Reply:
Thanks for catching that ! It was done when trying to change / access the `eventViewModel` and should not have stayed !